endpoints.json

```diff
@@ -140,5 +140,134 @@
 				}
 			]
 		}
-	}
-}
+	},
+
+"GET /api/articles (topic query)": {
+		"description": "serves an array of all articles that are on a specified topic",
+		"queries": ["topic"],
+		"exampleResponse": {
+			"articles":  [
+				{
+					"article_id": 3,
+					"author": "icellusedkars",
+					"title": "Eight pug gifs that remind me of mitch",
+					"topic": "mitch",
+					"created_at": "2020-11-03T09:12:00.000Z",
+					"votes": 0,
+					"article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+					"comment_count": 2
+				},
+				{
+					"article_id": 6,
+					"author": "icellusedkars",
+					"title": "A",
+					"topic": "mitch",
+					"created_at": "2020-10-18T01:00:00.000Z",
+					"votes": 0,
+					"article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
```

Comment:
I would find a way to not be so detailed with your examples - you just need to show what form a response will take, not include all the data!